src/containers/change-password/ChangePasswordModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback, useEffect } from 'react'
+
+import { Modal } from '@audius/stems'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { ChangePassword } from './ChangePassword'
+import { getCurrentPage } from './store/selectors'
+import { changePage, Page } from './store/slice'
+
+const messages = {
+  title: (
+    <>
+      <i className='emoji lock'></i>&nbsp;Change Password
+    </>
+  )
+}
+
+export const ChangePasswordModal = (props: any) => {
+  const { showModal, onClose } = props
+
+  const dispatch = useDispatch()
+
+  const currentPage = useSelector(getCurrentPage)
+  const allowClose = [
+    Page.CONFIRM_CREDENTIALS,
+    Page.FAILURE,
+    Page.SUCCESS
+  ].includes(currentPage)
+
+  useEffect(() => {
+    if (showModal) {
+      dispatch(changePage(Page.CONFIRM_CREDENTIALS))
+    }
+  }, [dispatch, showModal])
+
+  return (
+    <Modal
+      title={messages.title}
+      showTitleHeader
+      showDismissButton={allowClose}
+      dismissOnClickOutside={allowClose}
```

Comment:
I was still able to hit Escape to close the modal on the NEW_PASSWORD page

Reply:
yes lol -- unrelated bug we def need to fix

Reply:
I ask because I wonder if the other bug leads to this one - for me the first escape brings up the other modal, and the second escape closes both of them

Reply:
same